src/vs/platform/environment/common/environment.ts

```diff
@@ -8,60 +8,64 @@ import { createDecorator } from 'vs/platform/instantiation/common/instantiation'
 export interface ParsedArgs {
 	[arg: string]: any;
 	_: string[];
-	'folder-uri'?: string | string[];
-	_urls?: string[];
-	help?: boolean;
-	version?: boolean;
-	status?: boolean;
-	wait?: boolean;
-	waitMarkerFilePath?: string;
-	diff?: boolean;
+	aad?: boolean;
```

Comment:
Please add a code block such as `// {{SQL CARBON EDIT}}` around each block of code modified under `vs` folder.  This helps to make sure our patches to VS Code don't get lost during source code refreshes.

Also, there are a lot of modifications to this file.  Are they all necessary?  We try to minimize changes under `vs` to make source code refresh merge conflicts simpler.  Please prefer to augment the existing code rather than removing "unneeded" code.

Reply:
I had sorted a couple lists but can undo that. I'll add the comment block.